src/main/java/taxi/dao/jdbc/CarDaoJdbcImpl.java

```diff
@@ -0,0 +1,198 @@
+                cars.add(currentCar);
+            }
+            return cars;
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't get all elements "
+                    + "from car DB", ex);
+        }
+    }
+
+    @Override
+    public Car update(Car car) {
+        String queryForCars = "UPDATE car "
+                + "SET manufacturer_id = ?, model = ? "
+                + "WHERE car_id = ? AND deleted = false ";
+        try (Connection connection = ConnectionUtil.getConnection();
+                   PreparedStatement statementForCars = connection.prepareStatement(queryForCars)) {
+            statementForCars.setLong(1, car.getManufacturer().getId());
+            statementForCars.setString(2, car.getModel());
+            statementForCars.setLong(3, car.getId());
+            statementForCars.executeUpdate();
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't update car " + car.getModel(), ex);
+        }
+        deleteObjectInCarDriverById(car.getId());
+        for (int i = 0; i < car.getDrivers().size(); i++) {
+            insertNewDateInCarDriver(car.getDrivers().get(i).getId(), car.getId());
+        }
+        return car;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE car SET deleted = true WHERE car_id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            return statement.executeUpdate() > 0;
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't delete element with id "
+                    + id + " from DB", ex);
+        }
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String query = "SELECT cd.car_id "
+                + "FROM car_driver cd "
+                + "INNER JOIN driver d "
+                + "ON cd.driver_id = d.driver_id "
+                + "WHERE d.driver_id = ? AND deleted = false ";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, driverId);
+            ResultSet resultSet = statement.executeQuery();
+            List<Car> cars = new ArrayList<>();
+            while (resultSet.next()) {
+                cars.add(createCarFromDb(resultSet));
+            }
+            return cars;
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't get list car by driver id = "
+                    + driverId, ex);
+        }
+    }
+
+    private Car createCarFromDb(ResultSet resultSet) throws SQLException {
+        Long id = resultSet.getObject("car_id", Long.class);
+        Manufacturer manufacturer = ManufacturerDaoJdbcImpl.createManufacturerFromDb(resultSet);
```

Comment:
that`s optional but I think that using ManufacturerDaoJdbcImpl is not a good idea. Better to make a separate method here.